src/features/device-logs/lib/store.ts

```diff
@@ -220,6 +220,18 @@ function handleStreamingWrite(
 		req.pipe(parser);
 	}
 
+	// NOTE: this is testing code to test logging behavior with cloudflare
+	// these timers need to be defined in a non-harcoded way
+
+	// Wait for the client to finish the grace period and connect the buffers
+	setTimeout(ping, 5 * 1000);
+
+	function ping() {
+		// Send a single byte back every 10 seconds
+		res.write(new Uint8Array([1]));
```

Comment:
I think creating a buffer directly probably makes more sense, ie
```suggestion
		res.write(Buffer.alloc(1, 1));
```
which creates a size 1 buffer and pre-allocates it with the value 1 (default is 0): https://nodejs.org/docs/latest-v18.x/api/buffer.html#static-method-bufferallocsize-fill-encoding